branchff

```diff
@@ -55,6 +55,18 @@ PROG=${0##*/}
 usage=${1:-yes}
 
 source $(dirname $(readlink -ne $BASH_SOURCE))/lib/common.sh
+
+# Check prerequisites
+
+PREREQUISITE_TOOLS=("git" "jq" "go" "make")
+for PREREQ in ${PREREQUISITE_TOOLS[@]}; do
+  command -v $PREREQ > /dev/null 2>&1 || common::exit 1 \
+                                         "$PREREQ not installed"
+done;
+
+[[ -w /usr/local ]] || common::exit 1 "/usr/local is not writable."
+[[ $EUID -eq 0 ]]    || common::exit 1 "Please run as root!"
```

Comment:
this seems not necessary

Reply:
/cc @tpepper 
the [pre-requirements doc](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/branch-manager/README.md#pre-requirements) author

Reply:
I was thinking similar. Should I just check the EUID as this implies write access to /usr/local? I like this change you suggest a lot and think it makes things clearer.

Reply:
I was thinking of leaving the write access there :) since it's what the following steps needed, but it's not necessarily be run as root?

Reply:
This works too (I actually don't know the strict requirements yet--but I'd love to learn more!). I will take out the EUD portion. Thanks again this has been so cool and a great experience.

Reply:
I made the changes and pushed to my master branch :)